com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfRobotExtensions.cs

```diff
@@ -174,6 +174,7 @@ private static void ImportPipelinePostCreate(ImportPipelineData im)
         /// <returns></returns>
         public static IEnumerator<GameObject> Create(string filename, ImportSettings settings, bool loadStatus = false, bool forceRuntimeMode = false)
         {
+            UrdfGeometryCollision.BeginNewUrdfImport();
```

Comment:
It might be an overshoot refactoring, but I feel it might better to use instance instead of class (`static`), e.g. `var geometryCollision = new UrdfGeometryCollision()`?